src/libs/actions/Report.js

```diff
@@ -650,9 +650,7 @@ function fetchActions(reportID) {
  * @param {Number} reportID
  */
 function fetchInitialActions(reportID) {
-    Onyx.set(`${ONYXKEYS.COLLECTION.IS_LOADING_INITIAL_REPORT_ACTIONS}${reportID}`, true);
-    fetchActions(reportID)
-        .finally(() => Onyx.set(`${ONYXKEYS.COLLECTION.IS_LOADING_INITIAL_REPORT_ACTIONS}${reportID}`, false));
+    fetchActions(reportID);
```

Comment:
Now that this is all this method does, could you please just remove the method `fetchInitialActions()` and call `fetchActions()` directly?

Reply:
Just a note that it's going to be removed entirely in the next round of PRs.

Reply:
I can still replace it though, it's not that much work I think